.github/workflows/docs-mgmt-api-update.yml

```diff
@@ -11,31 +11,37 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v4
-      with:
-        ref: master
-        sparse-checkout: |
-          apps/docs
-
-    - name: Use Node.js
-      uses: actions/setup-node@v4
-      with:
-        node-version-file: '.nvmrc'
-        cache: 'npm'
-
-    - name: Install deps
-      run: npm ci
-
-    - name: Change to apps/docs/spec directory and run make command
-      working-directory: apps/docs/spec
-      run: make download.api.v1 dereference.api.v1 generate.sections.api.v1 format
-
-    - name: Create pull request
-      uses: peter-evans/create-pull-request@v6
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-        commit-message: 'feat: update mgmt api docs'
-        title: 'feat: update mgmt api docs'
-        body: 'This PR updates mgmt api docs automatically.'
-        branch: 'gha/auto-update-mgmt-api-docs'
-        base: 'master'
+      - uses: actions/checkout@v4
+        with:
+          ref: master
+          sparse-checkout: |
+            apps/docs
+
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          version: 9.10.0
```

Comment:
We shouldn't have the version here. According to the [docs](https://github.com/pnpm/action-setup?tab=readme-ov-file#version), it'll grab it from the package.json.